src/server-dev.ts

```diff
@@ -7,16 +7,19 @@ process.nextTick(async () => {
     try {
         await start(
             createConfig({
-                db: {
-                    user: 'unleash_user',
-                    password: 'password',
-                    host: 'localhost',
-                    port: 5432,
-                    database: process.env.UNLEASH_DATABASE_NAME || 'unleash',
-                    schema: process.env.UNLEASH_DATABASE_SCHEMA,
-                    ssl: false,
-                    applicationName: 'unleash',
-                },
+                db: process.env.DATABASE_URL
+                    ? undefined
```

Comment:
This way we can send DATABASE_URL instead of what we're doing currently. The problem with our current solution is that this db object always takes precedence